src/queue.c

```diff
@@ -195,6 +195,18 @@ mpd_send_add_id_to(struct mpd_connection *connection, const char *uri,
 	return mpd_send_s_u_command(connection, "addid", uri, to);
 }
 
+bool
+mpd_send_add_id_whence(struct mpd_connection *connection, const char *uri,
+		   unsigned to, enum mpd_position_whence whence)
+{
+	const char *whence_s = mpd_position_whence_char(whence);
+
+	char to_str[64] = "";
```

Comment:
What's the point of the initializer? It will be overwritten in the next line, so this initializer just adds pointless overhead.